### PR TITLE
Add Media Frame size function and corresponding tests

### PR DESCRIPTION
**Pull request by dbudniko**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

``` 
force_builders=Custom,Custom Win,Custom Mac
build_gapi_standalone:Linux x64=ade-0.1.1f
build_gapi_standalone:Win64=ade-0.1.1f
build_gapi_standalone:Mac=ade-0.1.1f
build_gapi_standalone:Linux x64 Debug=ade-0.1.1f

build_image:Custom=centos:7
buildworker:Custom=linux-1
build_gapi_standalone:Custom=ade-0.1.1f

Xbuild_image:Custom=ubuntu-openvino-2020.3.0:16.04
Xbuild_image:Custom Win=openvino-2020.3.0
Xbuild_image:Custom Mac=openvino-2020.3.0

test_modules:Custom=gapi
test_modules:Custom Win=gapi
test_modules:Custom Mac=gapi

buildworker:Custom=linux-1
// disabled due high memory usage: test_opencl:Custom=ON
test_opencl:Custom=OFF
test_bigdata:Custom=1
test_filter:Custom=*
```

### CODE REVIEW

File: modules/gapi/test/common/gapi_core_tests_inl.hpp [hunk 1]
```diff
@@ -1906,6 +1906,42 @@ TEST_P(SizeRTest, ParseTest)
...
+
+TEST_P(SizeMFTest, ParseTest)
+{
+    cv::GFrame in;
+    cv::Size out_sz;
+    int counter = 0;
+    cv::Mat bgr = cv::Mat::eye(sz.height, sz.width, CV_8UC3);
+    cv::MediaFrame frame = cv::MediaFrame::Create<TestMediaBGR>(bgr, [&counter]() {counter++; });
+
+    auto out = cv::gapi::streaming::size(in);
```
Comment (AsyaPronina) [inline modules/gapi/test/common/gapi_core_tests_inl.hpp:1938]: Could you please move `cv::GFrame` declaration a bit down to the 1937 line?

File: modules/gapi/test/common/gapi_core_tests_inl.hpp [hunk 2]
```diff
@@ -1906,6 +1906,42 @@ TEST_P(SizeRTest, ParseTest)
...
+    };
+};
+
+TEST_P(SizeMFTest, ParseTest)
+{
+    cv::GFrame in;
+    cv::Size out_sz;
+    int counter = 0;
+    cv::Mat bgr = cv::Mat::eye(sz.height, sz.width, CV_8UC3);
+    cv::MediaFrame frame = cv::MediaFrame::Create<TestMediaBGR>(bgr, [&counter]() {counter++; });
```
Comment (AsyaPronina) [inline modules/gapi/test/common/gapi_core_tests_inl.hpp:1936]: why do you pass such callback? what is the intent?

File: modules/gapi/test/common/gapi_core_tests_inl.hpp (hunk 1 above)
Comment (dbudniko) [reply modules/gapi/test/common/gapi_core_tests_inl.hpp:1938]: Done

File: modules/gapi/src/backends/cpu/gcpucore.cpp [hunk 3]
```diff
@@ -692,6 +692,16 @@ GAPI_OCV_KERNEL(GCPUSizeR, cv::gapi::streaming::GSizeR)
     }
 };
 
+GAPI_OCV_KERNEL(GCPUSizeMF, cv::gapi::streaming::GSizeMF)
+{
+    static void run(const cv::MediaFrame& in, cv::Size& out)
+    {
+        cv::GFrameDesc desc = in.desc();
```
Comment (rgarnov) [inline modules/gapi/src/backends/cpu/gcpucore.cpp:699]: out = in.desc().size;

**Review by rgarnov — CHANGES_REQUESTED** [review]

File: modules/gapi/src/backends/cpu/gcpucore.cpp (hunk 3 above)
Comment (dbudniko) [reply modules/gapi/src/backends/cpu/gcpucore.cpp:699]: Done

File: modules/gapi/test/common/gapi_core_tests_inl.hpp (hunk 2 above)
Comment (dbudniko) [reply modules/gapi/test/common/gapi_core_tests_inl.hpp]: Done

**Review by rgarnov — APPROVED** [review]

**Pull request merged**